crypto/evp/ctrl_params_translate.c

```diff
@@ -1642,6 +1642,72 @@ static int get_payload_public_key(enum state state,
     return ret;
 }
 
+static int get_payload_public_key_ec(enum state state,
+                                     const struct translation_st* translation,
+                                     struct translation_ctx_st* ctx)
+{
+    EVP_PKEY* pkey = ctx->p2;
+    unsigned char* buf = NULL;
+    BIGNUM* bn = NULL;
+    int ret = 0;
+
+    ctx->p2 = NULL;
+    switch (EVP_PKEY_get_base_id(pkey)) {
+#ifndef OPENSSL_NO_EC
+        case EVP_PKEY_EC:
+            if (ctx->params->data_type != OSSL_PARAM_UNSIGNED_INTEGER)
+                return 0;
+
+            const EC_KEY* eckey = EVP_PKEY_get0_EC_KEY(pkey);
```

Comment:
Use the same code as above... i.e. variables should only appear after a {

Reply:
ah ok. I see now.